oap-server/server-starter/src/main/resources/ui-initialized-templates/virtual_cache/virtual-cache-root.json

```diff
@@ -0,0 +1,122 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+[
+  {
+    "id": "Virtual-Cache-Root",
+    "configuration": {
+      "children": [
+        {
+          "x": 0,
+          "y": 2,
+          "w": 24,
+          "h": 52,
+          "i": "0",
+          "type": "Widget",
+          "widget": {
+            "title": "Virtual Cache"
+          },
+          "graph": {
+            "type": "ServiceList",
+            "dashboardName": "Virtual-Cache-Service",
+            "fontSize": 12,
+            "showXAxis": false,
+            "showYAxis": false,
+            "showGroup": false
+          },
+          "metrics": [
+            "cache_read_resp_time",
+            "cache_read_sla",
+            "cache_read_cpm",
+            "cache_write_resp_time",
+            "cache_write_sla",
+            "cache_write_cpm"
+          ],
+          "metricTypes": [
+            "readMetricsValues",
+            "readMetricsValues",
+            "readMetricsValues",
+            "readMetricsValues",
+            "readMetricsValues",
+            "readMetricsValues"
+          ],
+          "moved": false,
+          "metricConfig": [
+            {
+              "unit": "ms",
+              "label": "Read Latency",
+              "calculation": "average"
+            },
+            {
+              "label": "Successful Read Rate",
+              "unit": "%",
+              "calculation": "percentageAvg"
+            },
+            {
+              "label": "Read Traffic",
+              "unit": "calls / min",
+              "calculation": "average"
+            },
+            {
+              "unit": "ms",
+              "label": "Write Latency",
+              "calculation": "average"
+            },
+            {
+              "label": "Successful Write Rate",
+              "unit": "%",
+              "calculation": "percentageAvg"
+            },
+            {
+              "label": "Write Traffic",
+              "unit": "calls / min",
+              "calculation": "average"
+            }
+          ]
+        },
+        {
+          "x": 0,
+          "y": 0,
+          "w": 24,
+          "h": 2,
+          "i": "100",
+          "type": "Text",
+          "metricTypes": [
+            ""
+          ],
+          "metrics": [
+            ""
+          ],
+          "graph": {
+            "fontColor": "blue",
+            "backgroundColor": "white",
+            "content": "Observe the Virtual Cache which is conjectured by language agent through various plugins.",
+            "fontSize": 14,
+            "textAlign": "left",
+            "url": "https://skywalking.apache.org/docs/main/latest/en/setup/service-agent/virtual-cache/"
+          },
+          "moved": false
+        }
+      ],
+      "id": "Virtual-Cache-Root",
+      "layer": "VIRTUAL_CACHE",
+      "entity": "All",
+      "name": "Virtual-Cache-Root",
+      "isRoot": true
```

Comment:
I don't think UI supports this layer as root. Could you provide your local screenshot?

Reply:
![image](https://user-images.githubusercontent.com/3917424/190864174-1524406d-3ebb-40b8-8304-3736461db620.png)
![image](https://user-images.githubusercontent.com/3917424/190864183-1246675e-2c2c-4ac6-bb3f-fccf37ac9107.png)


I would submit PR to UI repo later

Reply:
![image](https://user-images.githubusercontent.com/3917424/190867959-8fd316c6-bd3e-48a6-86b0-1ec332390f7e.png)

![image](https://user-images.githubusercontent.com/3917424/190867977-ae5cf711-9600-473b-a000-8d99b899fbd1.png)


I  would submit PR to UI repo later